cli/cmd/plugin/unmanaged-cluster/cmd/configure.go

```diff
@@ -37,6 +37,9 @@ func init() {
 	ConfigureCmd.Flags().StringVar(&co.servicecidr, "service-cidr", "", "The CIDR to use for Service IP addresses. Default and format is '10.96.0.0/16'")
 	ConfigureCmd.Flags().Bool("tty-disable", false, "Disable log stylization and emojis")
 	ConfigureCmd.Flags().StringSliceVar(&co.additionalRepo, "additional-repo", []string{}, "Addresses for additional package repositories to install")
+	ConfigureCmd.Flags().StringSliceVar(&co.profile, "profile", []string{}, "(experimental) A profile to install. May be specified multiple times. Should be the fully qualified package name or a prefix to a package name found in an installed package repository. Profile mappings supported - profile-name:profile-version:profile-config-file")
+	ConfigureCmd.Flags().StringSliceVar(&co.profileConfigPath, "profile-config-file", []string{}, "(experimental) Optional: path to a profile values yaml file. Uses default values (when available) if not provided. May be specified multiple times. Strings given via this flag are ordered in a queue and are enqueud in the order they are specified and dequeud when missing profile configs are encountered.")
```

Comment:
typo in `enqueued` and `dequeued`.

@jpmcb is there an simpler way to say:

"Strings given via this flag are ordered in a queue and are enqueud in the order they are specified and dequeud when missing profile configs are encountered."

I feel like users might not have any clue what we're talking about.

Reply:
> I feel like users might not have any clue what we're talking about.

You're right, it is confusing. I couldn't really come up with a better way of saying "order won't matter when these flags are provided" and yet wanted to specify _how_ someone could order their flags.

This is partly a symptom of how cobra consumes these and actually is a long standing design problem in cobra around ordering flags.

A few thoughts:
- Just say "ordering doesn't work: use a config file"
- Recommend using config files or the full profile-mappings for multiple profiles
- Come up with a better way to describe this ordering.

Reply:
Planning to remove the references to queues: https://github.com/vmware-tanzu/community-edition/pull/3827#discussion_r840907023